backend/services/esweService.ts

```diff
@@ -105,72 +105,102 @@ export default class EsweService {
+      const createSkillAssessmentSummary = (learnerAssessment: curious.LearnerAssessment) => {
+        const { qualification, establishmentName } = learnerAssessment || {}
+        const { qualificationType, qualificationGrade, assessmentDate } = qualification || {}
 
-    const createSkillAssessmentSummary = (learnerAssessment: curious.LearnerAssessment) => {
-      const { qualification, establishmentName } = learnerAssessment || {}
-      const { qualificationType, qualificationGrade, assessmentDate } = qualification || {}
+        if (!assessmentDate) {
+          return [
+            {
+              label: qualificationType === AssessmentQualificationType.English ? 'English/Welsh' : qualificationType,
+              value: 'Awaiting assessment',
+            },
+          ]
+        }
 
-      if (!assessmentDate) {
         return [
           {
-            label: qualificationType,
-            value: 'Awaiting assessment',
+            label: qualificationType === AssessmentQualificationType.English ? 'English/Welsh' : qualificationType,
+            value: qualificationGrade,
+          },
+          {
+            label: 'Assessment date',
+            value: readableDateFormat(assessmentDate, CURIOUS_DATE_FORMAT),
+          },
+          {
+            label: 'Assessment location',
+            value: establishmentName,
           },
         ]
       }
 
-      return [
-        {
-          label: qualificationType === AssessmentQualificationType.English ? 'English/Welsh' : qualificationType,
-          value: qualificationGrade,
-        },
-        {
-          label: 'Assessment date',
-          value: readableDateFormat(assessmentDate, CURIOUS_DATE_FORMAT),
-        },
-        {
-          label: 'Assessment location',
-          value: establishmentName,
-        },
-      ]
-    }
-
-    const functionalSkillLevels = {
-      english: createSkillAssessmentSummary(englishGrade),
-      maths: createSkillAssessmentSummary(mathsGrade),
-      digiLit: createSkillAssessmentSummary(digitalLiteracyGrade),
+      skillLevels = {
+        english: createSkillAssessmentSummary(englishGrade),
+        maths: createSkillAssessmentSummary(mathsGrade),
+        digiLit: createSkillAssessmentSummary(digitalLiteracyGrade),
+      }
+    } catch (e) {
+      if (e.response?.body?.errorCode === 'VC500') {
+        log.info(`Offender record not found in Curious.`)
+        skillLevels = {
+          english: [
+            {
+              label: 'English/Welsh',
+              value: 'Awaiting assessment',
+            },
+          ],
+          maths: [
+            {
+              label: 'Maths',
+              value: 'Awaiting assessment',
+            },
+          ],
+          digiLit: [
+            {
+              label: 'Digital Literacy',
+              value: 'Awaiting assessment',
+            },
+          ],
+        }
+      } else {
+        log.warn(`Failed to get latest learning assessments. Reason: ${e}`)
```

Comment:
Does this not need to be a log.error? I suspect we need to know when users are not seeing the data.